### PR TITLE
Bug 1186569 - Alert on data which != summary

### PR DESCRIPTION
**Pull request by MikeLing**
Here is what we can got now
vagrant ~/treeherder $ ./manage.py test_analyze_perf --server https://treeherder.mozilla.org --project 'mozilla-inbound' --time-interval 604800

project,platform,signature,series,testrun_id,push_timestamp,change,percent change,t-value,revision

mozilla-inbound,windows8-64,b44b932ff6ffa9e7f54b23e1c1ae3cc5456faed4,damp simple.jsdebugger.reload.DAMP e10s opt,19522,1441347649,3.65083333333,5.35561477714,7.46132127486,8b380feae2ae

mozilla-inbound,windows7-32,06b47ef0361390db71ed967bdb83cf2bd63b8636,ts_paint summary pgo,19523,1441353594,17.9166666667,1.96275333212,7.25399829154,3416308a1151

mozilla-inbound,windows7-32,445ec2a7b7382b8128323728b592fa0e1b6e1f1f,tp5o_scroll thepiratebay.org pgo,19522,1441347649,0.2,5.61009817672,12.1705603252,8b380feae2ae

mozilla-inbound,windows7-32,fb5a1b176a68fb01eb89fd16c3875ede30463471,tart simple-open-DPI1.error.TART pgo,19523,1441353594,1.15416666667,3.71752200988,7.03301788484,3416308a1151

mozilla-inbound,windows8-64,58af5f76631e7f264cc0ac53164369c0b0552889,tp5o_scroll digg.com opt,19522,1441347649,0.181666666667,4.8747763864,11.3578853441,8b380feae2ae

mozilla-inbound,windowsxp,2ba8b84d21663e73f5e309f7c36c2b3892db5eb3,tp5o_scroll xinhuanet.com pgo,19522,1441347649,0.120833333333,2.95676998369,7.43074564748,8b380feae2ae

mozilla-inbound,linux32,8e0f45945c1c253e82bfaf5144fe680806b3340a,tp5o homeway.com.cn opt,19579,1441421674,-19.3625,4.59389457867,9.58564286749,45f93f97b68d

mozilla-inbound,windows7-32,26049d938905576034d375a06fb9832897079656,tart simple-close-DPI1.error.TART e10s opt,19545,1441376059,0.9025,3.65409271881,7.04213524879,6f5c75b6c255

mozilla-inbound,windows7-32,784c457f093bb02fd7aa89c4be03d1791aeec3dc,tp5o stackoverflow.com e10s opt,19522,1441347649,19.1875,5.33974953618,18.0601758931,8b380feae2ae

mozilla-inbound,windowsxp,5cfd0cbae43b215ee363ad86a156bc74a791606c,tp5o youtube.com opt,19522,1441347649,11.6791666667,5.25181743236,13.4850037252,8b380feae2ae

mozilla-inbound,linux32,80d4a20f7e5dc7178081d66ee8f289a9f316d81f,v8_7 DeltaBlue e10s pgo,19407,1441135175,-17.4266666667,8.19403628384,54.4900550072,ec887e3f764f

[Review on Reviewable](https://reviewable.io/reviews/mozilla/treeherder/943)

### CODE REVIEW

File: treeherder/model/management/commands/test_analyze_perf.py [hunk 1]
```diff
@@ -74,10 +74,15 @@ def handle(self, *args, **options):
                 signature_data = pc.get_performance_signatures(
                     project, time_interval=options['time_interval'])
                 signatures = []
+                signatures_should_ignored = []
```
Comment (wlach) [inline treeherder/model/management/commands/test_analyze_perf.py]: Let's make this a set, so we don't have tons of duplicate elements. Also, let's call this `signatures_to_ignore`. So:

    signatures_to_ignore = set()

File: treeherder/model/management/commands/test_analyze_perf.py [hunk 2]
```diff
@@ -74,10 +74,15 @@ def handle(self, *args, **options):
...
                 # if doing everything, only handle summary series
                 for (signature, properties) in signature_data.iteritems():
+                    signatures.append(signature)
                     if 'subtest_signatures' in properties:
-                        signatures.append(signature)
+                        signatures_should_ignored = signatures_should_ignored + \
```
Comment (wlach) [inline treeherder/model/management/commands/test_analyze_perf.py]: If using a set you can simplify this to: `signatures_to_ignore.update(properties['subtest_signatures'])`

Comment (wlach) [inline treeherder/model/management/commands/test_analyze_perf.py]: Also we should have a comment here like '# Don't alert on subtests which have a summary'

**Comment by wlach** [comment]
This looks right. Could you post the output of test_analyze_perf after your changes?

**Comment by MikeLing** [comment]
Hi Will, here is the output: https://pastebin.mozilla.org/8845653

**Pull request merged**